### DISCUSSION TITLE
How to upgrade to ESLint 8

### DISCUSSION THREAD

**Discussion by erosman** [eslint/eslint#17746] [Q&A] [answered]
I installed ESLint but its version is 6.4.0
How do I upgrade to the latest version?

_Ubuntu 23.10_

**Comment by Rec0iL99** [comment, marked as answer]
Hi @erosman, just as you update other npm packages, you can also update ESLint.

```bash
# if already installed
npm uninstall eslint

npm install eslint@latest
```

**Reply by erosman** [reply]
Thank you. I am new to ESLint & nmp.
